packages/flame/lib/src/game/game.dart

```diff
@@ -209,6 +210,10 @@ abstract class Game {
   /// not be on the flutter widget tree anymore.
   void onDetach() {}
 
+  /// The root component of the component tree, if the game uses a Flame
+  /// Component System, or `null` otherwise.
+  ComponentTreeRoot? get fcsRoot => null;
```

Comment:
A little sad that FCS spills over into `Game`, is it not possible to avoid this?

Reply:
Hmm, I'll try to convert this into an interface.